features/wasm-bigint.yml

```
@@ -0,0 +1,6 @@
+name: WebAssembly BigInt to i64 conversion in JavaScript
+description: An extension to the WebAssembly JavaScript API for bidirectionally converting BigInts and 64-bit WebAssembly integer values.
```

Me stealing a bit from the spec:

> WebAssembly i64 values can be passed to and from JavaScript (via imported or exported globals, table get or set operations, function return values or arguments) as BigInt objects. 

```suggestion
description: Integer values pass bidirectionally between JavaScript and WebAssembly, converted to i64 values in WebAssembly or BigInt values in JavaScript.
```